web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -20,9 +20,30 @@
             style="text-align: center;"
             outlined
           >
+            <v-tooltip
+              top
+            >
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  v-if="currentDandiset !== null"
+                  icon
+                  v-bind="attrs"
+                  v-on="on"
+                  @click="currentDandiset ? copy(currentDandiset.dandiset.identifier) : ''"
+                >
+                  <v-icon
+                    color="white"
```

Comment:
Are you running a dark mode extension in your browser? In the preview, this copy button being white makes it hard to see, and seems like there's nothing there

Reply:
ah yes. I have it turned off for 8080 but didn't think about it since we are on 8085. I'll fix

Reply:
fixed in f173382357ff401eb728606139da8570b486d49b